src/main/java/com/example/bookstore/controller/users/HomePageController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.bookstore.controller.users;
+
+import com.example.bookstore.entity.Book;
+import com.example.bookstore.service.BooksService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+@Controller
+public class HomePageController {
```

Comment:
```suggestion
public class HomesController {
```
mà thực ra chị thích `StaticsPageController ` hơn á
các page như home, help. about. đểu dc khai báo action trolng controller ày là đẹp

Reply:
Dạ vâng e sẽ sửa